cmd/sync.go

```diff
@@ -533,6 +533,17 @@ func (cca *cookedSyncCmdArgs) process() (err error) {
 		return err
 	}
 
+	srcCredInfo, _, err := getCredentialInfoForLocation(ctx, cca.fromTo.From(), cca.source.Value, cca.source.SAS, true)
+
+	if err != nil {
+		return err
+	}
+
+	// If the source wants OAuth and the destination doesn't, override the credential type because this could be a download, or oauth to SAS.
```

Comment:
A bit confused here, how can we do Oauth -> SAS sync? (I don't think we ever merged in the identity SAS change, right?)

Reply:
@adreed-msft